duckduckgo/privacy-essentials-safari
# New icons

Task/Issue URL: https://app.asana.com/0/1144193759412024/1200007688579509/f
Tech Design URL: -
CC: @brindy 

**Description**:
Design team provided a new version of toolbar icons for each appearance mode.


**Steps to test this PR**:
1. Make sure toolbar icon displays proper grade on various website
2. Make sure new Dax icons are displayed on homepage.  Each appearance mode (dark/light) has its own.


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
